### ISSUE TITLE
Data import doubly decodes text.

### ISSUE THREAD

**Issue by jsi** [enonic/xp#7069]
When importing data that has been exported from cms, using the cms2xp tool, a string like "&amp;quot;" turns into an single quote: "  -  So, it seems the text is decoded twice, first to "&quot;" and then to the quote sign: "

Here is an example from a customer.  In cms, they have this custom HTML:
```
<p><a href="/naringsliv/tema/forsikrings-og-pensjonspakker" data-event="{&quot;event_category&quot;: &quot;button&quot;, &quot;event_action&quot;: &quot;click&quot;,&quot;event_label&quot;: &quot;se-php&quot;}">Se pakkene her</a></p>
```
This is also what the import should look like, when we import it to XP.

The cms2xp tool will encode the text to this:
```
<string name="button-link">&lt;p&gt;&lt;a data-event="{&amp;quot;event_category&amp;quot;: &amp;quot;button&amp;quot;, &amp;quot;event_action&amp;quot;: &amp;quot;click&amp;quot;,&amp;quot;event_label&amp;quot;: &amp;quot;se-php&amp;quot;}" href="/naringsliv/tema/forsikrings-og-pensjonspakker"&gt;Se pakkene her&lt;/a&gt;&lt;/p&gt;</string>
```
So, the text "&quot;" have been encoded to "&amp;quot;"  -  This is normal.  The ampersand (&) is a special character that should be encoded.  The problem arise when the text is importet to XP.  Then we get this:
```
"button-link": "<p><a data-event="{"event_category": "button", "event_action": "click","event_label": "se-php"}" href="/naringsliv/tema/forsikrings-og-pensjonspakker">Se pakkene her</a></p>",
```

**Please make sure the import only decodes the text once.**

**Comment by GlennRicaud** [comment]
To reproduce

Create a node/content with HTML encoded text stored inside a property
![Screenshot 2019-04-12 at 12 26 44](https://user-images.githubusercontent.com/12030113/56031280-48e75400-5d1f-11e9-8927-1ba79316ee76.png)

Export it
![Screenshot 2019-04-12 at 12 27 26](https://user-images.githubusercontent.com/12030113/56031297-54d31600-5d1f-11e9-9ea4-539faeb86a30.png)

Delete it and reimport the export
The property value has changed. It has been decoded...
![Screenshot 2019-04-12 at 12 28 44](https://user-images.githubusercontent.com/12030113/56031307-5c92ba80-5d1f-11e9-8ae8-48502140763d.png)

**Comment by GlennRicaud** [comment]
To be fixe din 6.15 and 7.0

**Comment by sigdestad** [comment]
Hmm.. is this error related to CMS2XP, or the XP import?

**Comment by GlennRicaud** [comment]
XP Import

**Comment by jsi** [comment]
Make sure this fix is also made on the 6.15 branch!
